Binary files pre/network-manager-applet.ORIG/network-manager-applet-0.8.6.0-0-x86_64.pkg.tar.xz and pre/network-manager-applet/network-manager-applet-0.8.6.0-0-x86_64.pkg.tar.xz differ
diff -ru pre/network-manager-applet.ORIG/src/network-manager-applet-0.8.6.0/src/connection-editor/nm-connection-list.c pre/network-manager-applet/src/network-manager-applet-0.8.6.0/src/connection-editor/nm-connection-list.c
--- pre/network-manager-applet.ORIG/src/network-manager-applet-0.8.6.0/src/connection-editor/nm-connection-list.c	2013-02-07 13:47:31.274708214 -0800
+++ pre/network-manager-applet/src/network-manager-applet-0.8.6.0/src/connection-editor/nm-connection-list.c	2013-02-07 14:00:20.598396760 -0800
@@ -1405,11 +1405,11 @@
 	                    wired_connection_new);
 
 	add_connection_tab (self, def_type, NM_TYPE_SETTING_WIRELESS,
-	                    self->wireless_icon, "wireless", _("Wireless"),
+	                    self->wireless_icon, "wireless", _("Wi-Fi"),
 	                    wifi_connection_new);
 
 	add_connection_tab (self, def_type, NM_TYPE_SETTING_GSM,
-	                    self->wwan_icon, "wwan", _("Mobile Broadband"),
+	                    self->wwan_icon, "wwan", _("Mobile"),
 	                    mobile_connection_new);
 
 	add_connection_tab (self, def_type, NM_TYPE_SETTING_VPN,
